incubator/kafka: Update to Confluent 5.4.1 containers

thebigcorporation opened the pull request:
- Update Kafka and schema-registry to 5.4.0
 - Update Chart.yaml and Documentation / README

Signed-off-by: Sven-Thorsten Dietrich <sdietrich@manetu.com>

<!--
Thank you for contributing to helm/charts. Before you submit this PR we'd like to
make sure you are aware of our technical requirements and best practices:

* https://github.com/helm/charts/blob/master/CONTRIBUTING.md#technical-requirements
* https://github.com/helm/helm/tree/master/docs/chart_best_practices

For a quick overview across what we will look at reviewing your PR, please read
our review guidelines:

* https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md

Following our best practices right from the start will accelerate the review process and
help get your PR merged quicker.

When updates to your PR are requested, please add new commits and do not squash the
history. This will make it easier to identify new changes. The PR will be squashed
anyways when it is merged. Thanks.

For fast feedback, please @-mention maintainers that are listed in the Chart.yaml file.

Please make sure you test your changes before you push them. Once pushed, a CircleCI
will run across your changes and do some initial checks and linting. These checks run
very quickly. Please check the results. We would like these checks to pass before we
even continue reviewing your changes.
-->
#### Is this a new chart
> NOTE: We're experiencing a high volume of PRs to this repo and reviews will be delayed. Please host your own chart repository and submit your repository to the Helm Hub instead of this repo to make them discoverable to the community. [Here](https://github.com/helm/hub/blob/master/Repositories.md) is how to submit new chart repositories to the Helm Hub.

#### What this PR does / why we need it:

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Special notes for your reviewer:

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

davidkarlsen commented on incubator/kafka/Chart.yaml:
```diff
@@ -3,7 +3,7 @@ description: Apache Kafka is publish-subscribe messaging rethought as a distribu
   commit log.
 name: kafka
 version: 0.20.7
```
```suggestion
version: 0.20.8
```

thebigcorporation replied:
Can do. 
Unsure of your approach here; with a 3 version roll-forward of Kafka it could also be 21.0?

thebigcorporation replied:
Updated with 20.8

davidkarlsen committed:
bump further

Signed-off-by: David J. M. Karlsen <david@davidkarlsen.com>

davidkarlsen commented on incubator/schema-registry/Chart.yaml:
```diff
@@ -1,7 +1,7 @@
 name: schema-registry
 home: https://docs.confluent.io/current/schema-registry/docs/index.html
-version: 1.1.8
```
schema reg update is unrelated to PR subject

thebigcorporation replied:
Rebased and pushed; updated to 5.4.1.
reg must've along with Kafka - these containers are built and released together
https://hub.docker.com/r/confluentinc/cp-schema-registry/tags

SvenDietrich committed:
Update to Confluent 5.4.1 containers

 - Update Kafka and schema-registry to 5.4.1
 - Update Chart.yaml and Documentation / README

Signed-off-by: Sven-Thorsten Dietrich <sdietrich@manetu.com>
